docs/DeveloperGuide.md

```diff
@@ -464,6 +464,113 @@ As the `Model` class was built prior to the implementation of this feature, we d
 
 This design allows for easy extension to accommodate future enhancements or additional search criteria. New prefixes can be added to support additional search criteria without significant changes as we merely need to update our `Predicate` logic. This ensures that the implementation remains adaptable to evolving requirements and we can upgrade and improve the feature whenever required.
 
+<br>
+
+### **Exam Score Features**
+
+There are 3 main commands that are used to interact with exam scores of each person: `addScore`, `editScore` and `deleteScore`.
+
+<br>
+
+#### **Add Score Command** : `addScore`
+
+The `addScore` command allows users to add a score for an exam to a person displayed in the application. 
+The user should select the exam they want to add a score for, then specify the index of the person they want to add a score for, and the score they want to add.
+The score is then stored in a hashmap named `scores` within the `Person` object in the `Model` component. 
+This hashmap maps the selected exam (an `Exam` object) to the specified score (a `Score` object).
+
+##### Implementation Details
```

Comment:
Seems unnecessary since you will be elaborating on this further in the next section, which makes this part somewhat repetitive.